dat-schema/_Core.gql

```diff
@@ -5348,6 +5385,47 @@ type AtlasMemoryLine {
   PathArt: string
 }
 
+# Added 3.23
+type AzmeriCounterQuests {
+  _: string
+  _: rid
+  _: rid
+  _: rid
+  _: i32
+}
+
+# Added 3.23
+type AzmeriFeatureRooms {
+  _: i32
+  _: i32
+  _: string
+  _: string
+  _: [_]
+  _: bool
+}
+
+# Added 3.23
+type azmerilifescalingperlevel {
+  _: i32
+  _: i32
+}
+
+# Added 3.23
+type azmeriwoodsdusttype {
```

Comment:
This is league mechanic table, so should go into 3.23 file + should be properly CamelCased.

Reply:
done